docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+services:
+  node_dev_env:
+    restart: always
+    build: .
+    container_name: wolfsbane-backend
+    ports:
+      - 5000:3000
```

Comment:
Please change the port to `3000:3000`. There is no need to have different ports.

Reply:
ok